action.yml

```diff
@@ -13,7 +13,7 @@ inputs:
   changelog:
     description: Set true to create a changelog-X.md file (X is the version)
     required: false
-    default: false
+    default: "false"
```

Comment:
why is this needed?

Reply:
![image](https://user-images.githubusercontent.com/4209558/182365441-82e2a9f5-979c-40ba-a293-565bf690d4ed.png)

Auto formatted, I've disabled auto formatting in the workspace now (these commits were done before merging previous PR on tomerfi/version-bumper)